src/renderer/store/AuthStore.ts

```diff
@@ -0,0 +1,20 @@
+import { Authentication } from "@xmcl/user";
+
+import Store from "electron-store";
+
+interface AuthStoreData extends Authentication{
+	loggedIn: true, 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=5164335)